api-reference/10 UI Components/dxDropDownBox/1 Configuration/fieldTemplate.md

```diff
@@ -3,27 +3,34 @@ id: dxDropDownBox.Options.fieldTemplate
 type: template
 default: null
 ---
+
 ---
```

Comment:
I guess that somethings wrong here 
![image](https://user-images.githubusercontent.com/33040934/111816982-803ce300-88ee-11eb-99b6-30c714b88962.png)